test/integration/kubernetes_deploy_test.rb

```diff
@@ -1095,7 +1103,7 @@ def test_jobs_can_fail
       "Result: FAILURE",
       "Job/hello-job: FAILED",
       "Final status: Failed",
-      %r{\[Job/hello-job\]	DeadlineExceeded: Job was active longer than specified deadline \(\d+ events\)},
+      %r{\[Job/hello-job\]\tDeadlineExceeded: Job was active longer than specified deadline \(\d+ events\)},
```

Comment:
This (and the other change like this) were to satisfy the linter, which was complaining about the tab literal